### PR TITLE
Add RenameMethodCallRector, fixes #1210

### LINKED ISSUE #1210: Rename MethodCall Rector
The current RenameMethodRector renames also the ClassMethod itself. So this functionality is unuseable when the method is deprecated.

I could make it configureable :) if wanted

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

File: tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture3.php.inc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<?php
+
+namespace Rector\Tests\Rector\MethodCall\RenameMethodCallRector\Fixture;
+
+use Nette\Utils\Html;
+
+class SomeClass4
+{
+    private function createHtml()
+    {
```
Comment (TomasVotruba) [inline tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture3.php.inc]: Could also add rename for `createHtml`? So we know the `ClassMethod` is untouched

**Comment by TomasVotruba** [comment]
Great work, thank you for detailed tests :+1:

**Comment by shyim** [comment]
I added a another test case

File: tests/Rector/MethodCall/RenameMethodCallRector/RenameMethodCallRectorTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+final class RenameMethodCallRectorTest extends AbstractRectorTestCase
+{
+    public function test(): void
+    {
+        $this->doTestFiles([
+            __DIR__ . '/Fixture/fixture.php.inc',
+            __DIR__ . '/Fixture/fixture2.php.inc',
+            __DIR__ . '/Fixture/fixture3.php.inc',
+        ]);
```
Comment (TomasVotruba) [inline tests/Rector/MethodCall/RenameMethodCallRector/RenameMethodCallRectorTest.php]: `fixture4.php.inc` is missing here

File: tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture4.php.inc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+class SomeClass5
+{
+    private function createHtml()
+    {
+    }
+}
+
+?>
+-----
```
Comment (TomasVotruba) [inline tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture4.php.inc]: When there should be no change, you can keep only content above `-----`

**Comment by TomasVotruba** [comment]
Thanks!

Please keep commits separated until before the merge. When you force push, I don't see the new changes, but all code at once. It's harder to review

**Comment by shyim** [comment]
Sorry 🙈 Fixed that one :)

File: tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture4.php.inc [hunk 4]
```diff
@@ -12,18 +12,4 @@ class SomeClass5
...
-
-class SomeClass5
-{
-    private function createHtml()
-    {
-    }
-}
-
-?>
+-----
```
Comment (TomasVotruba) [inline tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture4.php.inc]: This line should be remove too. It's separator that indicates to compare to content after it.
Now it's compared to empty string, which fails

```
before|after
stay_the_same
```

**Comment by shyim** [comment]
Ahh okay 🙈  fixed :)

**Comment by TomasVotruba** [comment]
Awesome work :+1:  It's ready.

Please squash into 1 commit and I'll merge it

**Comment by shyim** [comment]
Github has a Squash and merge button 😅

**Comment by TomasVotruba** [comment]
I know, but it sucks :) 

Squash is for the PR, that's where we go to get more intell on the feature.

**Pull request merged**